src/idl_gen_rust.cpp

```diff
@@ -352,6 +352,7 @@ class RustGenerator : public BaseGenerator {
   }
 
   bool generate() {
+    code_.SetValue("ACCESS_TYPE", "pub");
```

Comment:
I think this is no longer necessary and should be removed.

This is why I'm not a huge fan of the CodeWriter abstraction, it encourages stateful action at a difference :(